ci/nightly/pipeline.template.yml

```diff
@@ -811,6 +811,17 @@ steps:
               composition: platform-checks
               args: [--scenario=TogglePersistRoundtripSpine, "--seed=$BUILDKITE_JOB_ID"]
 
+      - id: checks-toggle-persist-use-arrow-rs-library
```

Comment:
Let's also add the non-default value of this dyncfg to https://github.com/MaterializeInc/materialize/blob/611fb8f3775552529548578418753cc532f426ba/src/persist-proc/src/lib.rs#L84-L90

Reply:
Unfortunately the current framework only allows toggling a single dyncfg :(

Reply:
Oh, sorry, I was being unclear! I probably could have picked a less confusing place to hang this comment off of :)

The ask is for you to add something like the following to src/persist-proc/src/lib.rs. It will cause all the persist unit tests to also run with both the old and new libraries. 

```
                {
                    // New arrow/parquet library
                    let mut x = ::mz_dyncfg::ConfigUpdates::default();
                    x.add_dynamic("persist_use_arrow_rs_library_read", ::mz_dyncfg::ConfigVal::Bool(true));
                    x.add_dynamic("persist_use_arrow_rs_library_write", ::mz_dyncfg::ConfigVal::Bool(true));
                    x
                },
```

Reply:
Ah ha! I should have realized that, will update :)

Reply:
> Unfortunately the current framework only allows toggling a single dyncfg :(

Is the downside that you need to create a new separate scenario for this var or that you cannot change both `persist_roundtrip_spine` and the new one at once? If the latter is the case, I am sure I can adjust the code to make that possible.

Reply:
You could consider adding the variable here as well: https://github.com/MaterializeInc/materialize/blob/1dd47ee530807494c7f03babfe9274f5c050f7da/misc/python/materialize/parallel_workload/action.py#L907

It will randomly flip the flag while other operations happen concurrently in the parallel workload tests.

Reply:
I was hoping we would be able to randomly toggle an arbitrary number of flags, which it looks like is exactly what the `FlipFlagsAction` does! I missed that initially, will add it now

Reply:
> I was hoping we would be able to randomly toggle an arbitrary number of flags [...]

Addressed for the platform-checks with https://github.com/MaterializeInc/materialize/pull/26987.